### PR TITLE
Copy login command to clipboard, or manually from notification

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

This PR adds ability to user to get login command copied to clipboard, or if that's not possible a notification will appear where user can copy the command manually.

<details>
  <summary>Screenshots</summary>

  ##### Menu item

  Screenshot 2020-11-16 at 14 19 14

  <hr/>

  ##### Login command has been put to clipboard successfully

Screenshot 2020-11-16 at 15 20 33

  <hr/>

  ##### Failed to put login command to clipboard

Screenshot 2020-11-16 at 15 22 22

<hr/>

  ##### Copy login command manually

Screenshot 2020-11-16 at 15 22 29

</details>

### What issues does this PR fix or reference?

resolves https://github.com/eclipse/che/issues/13852

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]

File: src/app/navbar/navbar.controller.ts [hunk 1]
```diff
@@ -52,6 +57,12 @@ export class CheNavBarController {
         this.gotoProfile();
       }
     },
+    {
+      name: 'Copy Login Command',
```
Comment (sleshchenko) [inline src/app/navbar/navbar.controller.ts:61]: From the sprint review:

> @benoitf : Maybe say ‘chectl login command’ as I may think it’s an oc or kubectl command (but I’m not a UX expert)

I'm OK with either option. We made similar as OpenShift Console 3.x has.

Comment (sleshchenko) [reply src/app/navbar/navbar.controller.ts:61]: from UX: `Copy crwctl Login Command`/`Copy crwctl Login Command`

Comment (benoitf) [reply src/app/navbar/navbar.controller.ts:61]: there is one that should be chectl no ?

Comment (akurinnoy) [reply src/app/navbar/navbar.controller.ts:61]: sure, that's just a typo

**Review by olexii4 — APPROVED** [review]

**Pull request merged**